zerver/openapi/zulip.yaml

```diff
@@ -2560,21 +2560,11 @@ paths:
                           type: string
                           description: |
                             Whether the emoji has been deactivated or not.
-                        author:
+                        author_id:
+                          type: int
                           description: |
-                            An object describing the user who created the custom emoji,
+                            An id of the object describing the user who created the custom emoji,
                             with the following fields:
```

Comment:
Since author is no longer an object, so I think the "fields" line can be removed.

Reply:
Done!